code/datums/status_effects/rogue/roguebuff.dm

```diff
@@ -373,12 +373,12 @@
 /datum/status_effect/buff/trollshape
 	id = "trollshape"
 	alert_type = /atom/movable/screen/alert/status_effect/buff/trollshape
-	effectedstats = list(STATKEY_STR = 5, STATKEY_END = 2, STATKEY_SPD = -3, STATKEY_INT = -5)
+	effectedstats = list(STATKEY_STR = 4, STATKEY_END = 2, STATKEY_SPD = -3, STATKEY_INT = -5)
```

Comment:
is it already no rare enough with a -3 SPD debuff, considering you need now a head of troll and 2 horns, a briar alone won't solo a troll unless they somehow cheese or get help from someone and considering how hard it is you now , you give it a debuff? i don't understand it